txtdirect.go

```diff
@@ -167,5 +183,34 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 		return gometa(w, rec, host, path)
 	}
 
+	if rec.Type == "path" {
```

Comment:
This code would probably never be reached. Only the first request is type=path. After it traverses the wildcards and didn't find a type=host it should redirect to a default instead. This would mean that the return from getRecords is either type=host or a failure.